### DISCUSSION TITLE
Problem with Tesseract.js and deploy / install

### DISCUSSION THREAD

**Discussion by NeikiDev** [vercel/community#1706] [Help]
### Summary

Aborted(Error: ENOENT: no such file or directory, open '/var/task/node_modules/tesseract.js-core/tesseract-core-simd.wasm')
Uncaught Exception 	{"errorType":"RuntimeError","errorMessage":"Aborted(Error: ENOENT: no such file or directory, open '/var/task/node_modules/tesseract.js-core/tesseract-core-simd.wasm'). Build with -sASSERTIONS for more info.","stack":["RuntimeError: Aborted(Error: ENOENT: no such file or directory, open '/var/task/node_modules/tesseract.js-core/tesseract-core-simd.wasm'). Build with -sASSERTIONS for more info.","    at n (/var/task/node_modules/tesseract.js-core/tesseract-core-simd.js:18:375)","    at Ua (/var/task/node_modules/tesseract.js-core/tesseract-core-simd.js:19:317)","    at /var/task/node_modules/tesseract.js-core/tesseract-core-simd.js:20:394"]}
Unknown application error occurred
Runtime.Unknown

### Additional information

i got this error here and i tried to use the tesseract module with nodejs and npm

i got everything normal, and have the packages in my package.json,
my install command is "npm install --save" but it seems that it doesnt work!

**Comment by AChangXD** [comment]
Bumping this, seems like the .wasm files aren't included in the final build?

**Reply by AChangXD** [reply]
@amyegan not sure if you can take a look into this? I'm deploying a express.js backend and tesseract's .wasm files aren't being found in node_modules/tesseract.js-core. I'm assuming somewhere in the build process, those files were being purged?

**Comment by EthanQuarry** [comment]
Can we get some help @Vercel
